src/renderer/components/dock/terminal/get-terminal-api.injectable.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable, lifecycleEnum } from "@ogre-tools/injectable";
+import type { TerminalApi } from "../../../api/terminal-api";
+import { bind } from "../../../utils";
+import type { TabId } from "../dock/store";
+import type { TerminalStore } from "./store";
+import terminalStoreInjectable from "./store.injectable";
+
+interface Dependencies {
+  terminalStore: TerminalStore;
+}
+
+function getTerminalApi({ terminalStore }: Dependencies, tabId: TabId): TerminalApi {
+  return terminalStore.getTerminalApi(tabId);
+}
+
+const getTerminalApiInjectable = getInjectable({
+  instantiate: (di) => bind(getTerminalApi, null, {
```

Comment:
This bind pattern is just complexity which shouldn't be added. Configuration (e.g. dependencies) shouldn't be mixed with arguments. Arrow function would be much simpler.

Reply:
Inline `getTerminalApi`